src/main/java/com/targomo/client/api/request/config/builder/JSONBuilder.java

```diff
@@ -40,6 +43,18 @@ public static StringBuilder append(final StringBuilder builder, final String key
 		return builder.append("\"").append(key).append("\":").append(value).append(",");
 	}
 
+	/**
+	 * Appends a list of objects adding quotes and comma separated
+	 * @param builder Builder to append the value to
+	 * @param key Field name
+	 * @param list List of objects
+	 * @return Builder after appending
+	 */
+	public static <T> StringBuilder appendStringList(final StringBuilder builder, final String key, final List<T> list) {
+		return append(builder, key, list.stream().map(val -> "\"" + val.toString()  + "\"").collect(Collectors.toList()));
```

Comment:
why are we adding `"` at either side? This is normally done for Strings and getTargetGeohashes are Strings - this does not seem like an optimal solution

Reply:
I see. thx